default_settings.json

```diff
@@ -0,0 +1,5254 @@
+
+	// Controls whether the editor should highlight the active bracket pair.
+	"editor.guides.highlightActiveBracketPair": true,
+
+	// Controls whether the editor should highlight the active indent guide.
+	"editor.guides.highlightActiveIndentation": true,
+
+	// Controls whether the editor should render indent guides.
+	"editor.guides.indentation": true,
+
+	// Controls whether the cursor should be hidden in the overview ruler.
+	"editor.hideCursorInOverviewRuler": false,
+
+	// Prefer showing hovers above the line, if there's space.
+	"editor.hover.above": true,
+
+	// Controls the delay in milliseconds after which the hover is shown.
+	"editor.hover.delay": 300,
+
+	// Controls whether the hover is shown.
+	"editor.hover.enabled": true,
+
+	// Controls whether the hover should remain visible when mouse is moved over it.
+	"editor.hover.sticky": true,
+
+	// Enables the inlay hints in the editor.
+	"editor.inlayHints.enabled": true,
+
+	// Controls font family of inlay hints in the editor. When set to empty, the `editor.fontFamily` is used.
+	"editor.inlayHints.fontFamily": "",
+
+	// Controls font size of inlay hints in the editor. A default of 90% of `editor.fontSize` is used when the configured value is less than `5` or greater than the editor font size.
+	"editor.inlayHints.fontSize": 0,
+
+	// Controls whether to automatically show inline suggestions in the editor.
+	"editor.inlineSuggest.enabled": true,
+
+	// Insert spaces when pressing `Tab`. This setting is overridden based on the file contents when `editor.detectIndentation` is on.
+	"editor.insertSpaces": true,
+
+	// Defines the bracket symbols that increase or decrease the indentation.
+	"editor.language.brackets": false,
+
+	// Defines the bracket pairs that are colorized by their nesting level if bracket pair colorization is enabled.
+	"editor.language.colorizedBracketPairs": false,
+
+	// Special handling for large files to disable certain memory intensive features.
+	"editor.largeFileOptimizations": true,
+
+	// Controls the letter spacing in pixels.
+	"editor.letterSpacing": 0,
+
+	// Enables the code action lightbulb in the editor.
+	"editor.lightbulb.enabled": true,
+
+	// Controls the line height. 
+	//  - Use 0 to automatically compute the line height from the font size.
+	//  - Values between 0 and 8 will be used as a multiplier with the font size.
+	//  - Values greater than or equal to 8 will be used as effective values.
+	"editor.lineHeight": 0,
+
+	// Controls the display of line numbers.
+	//  - off: Line numbers are not rendered.
+	//  - on: Line numbers are rendered as absolute number.
+	//  - relative: Line numbers are rendered as distance in lines to cursor position.
+	//  - interval: Line numbers are rendered every 10 lines.
+	"editor.lineNumbers": "on",
+
+	// Controls whether the editor has linked editing enabled. Depending on the language, related symbols, e.g. HTML tags, are updated while editing.
+	"editor.linkedEditing": false,
```

Comment:
I think this should be `true`. Really handy in`.html` files, not working by default in `.html.erb` files — need to figure out how.

Reply:
Related: https://marketplace.visualstudio.com/items?itemName=formulahendry.auto-rename-tag